### PR TITLE
Fix german language specifications

### PR DESCRIPTION
**Pull request by Ben305**
Fix german language file: As per DIN 1333, the currency symbol is space separated. Also, abbreviations aren't used/known in Germany.

### CODE REVIEW

File: languages/de-DE.js [hunk 1]
```diff
@@ -27,10 +27,11 @@
         },
         currency: {
             symbol: '€',
-            position: 'postfix'
+            position: 'postfix',
+            spaceSeparated: true
         },
         defaults: {
-            currencyFormat: ',4 a'
+            currencyFormat: '0,0 $'
```
Comment (BenjaminVanRyseghem) [inline languages/de-DE.js]: are you sure about this?

it sounds a bit strange

Comment (Ben305) [inline languages/de-DE.js]: Well, the output that this format produces, seems right to my (I'm german):

1000 --> 1.000 €
9255 --> 9.255 €

Comment (BenjaminVanRyseghem) [inline languages/de-DE.js]: it was more about the non-average part of the format

Comment (Ben305) [inline languages/de-DE.js]: Can you explain what `,4` produces? Maybe then I can answer better.

Comment (BenjaminVanRyseghem) [inline languages/de-DE.js]: the `4` is (IIRC) to force to display 4 digits at max

Comment (Ben305) [inline languages/de-DE.js]: I've changed the currencyFormat to `,4 $`, which produces also correct outputs.

File: languages/de-DE.js [hunk 2]
```diff
@@ -27,10 +27,11 @@
         },
         currency: {
             symbol: '€',
-            position: 'postfix'
+            position: 'postfix',
+            spaceSeparated: true
         },
         defaults: {
-            currencyFormat: ',4 a'
+            currencyFormat: ',4 $'
```
Comment (BenjaminVanRyseghem) [inline languages/de-DE.js]: in `currencyFormat`, you shouldn't need a `$`

Also, why removing the `a`?

Comment (Ben305) [inline languages/de-DE.js]: You're right, I've removed the `$`

The `a` was removed because in Germany you'll never write 10,97 k € for 10.970 €

Comment (BenjaminVanRyseghem) [inline languages/de-DE.js]: ok :+1:

do you have a reference for it?

Comment (Ben305) [inline languages/de-DE.js]: There's a specification for writing numbers: DIN 1333. You can see some examples on Wikipedia: https://de.wikipedia.org/wiki/Schreibweise_von_Zahlen#Deutschland_und_.C3.96sterreich

**Comment by Ben305** [comment]
You can take a look at the angular locales, for example:

In `https://github.com/angular/angular.js/blob/master/src/ngLocale/angular-locale_de.js` you can see that the suffix for positive and negative numbers (`posSuf` and `negSuf`) is declared like this: 

`\u00a0\u00a4` (non-breaking space + currency sign)

I hope this is proof enough for the `spaceSeparated` change

**Comment by Ben305** [comment]
@BenjaminVanRyseghem What can I do to get this merged?

**Comment by BenjaminVanRyseghem** [comment]
@Ben305 sorry for the delay. It's mainly because we are in a rush at work that not much has changed here.
 
I will merge your fix and do my best to do a release as soon as I can

**Comment by BenjaminVanRyseghem** [comment]
in numbro 1.9.3

**Pull request merged**